docs/CloudEvents.md

```diff
@@ -405,7 +405,7 @@ The `PartitionedEvent` class defines four constructors:
 
 ## Protocol Bindings
 
-The `ProtocolBindings` static class defines the following bindings (corresponding to the [CloudEvents spec](https://github.com/cloudevents/specsharpcloudevents-documents)):
```

Comment:
I think it was trying to get this anchor - https://github.com/cloudevents/spec#cloudevents-documents

Reply:
nice work. fixed it